od-content-visibility.php

```diff
@@ -24,12 +24,13 @@
 }
 // @codeCoverageIgnoreEnd
 
-const OD_CONTENT_VISIBILITY_VERSION = '0.1.0';
+const OD_CONTENT_VISIBILITY_VERSION = '0.2.0';
```

Comment:
Update the `Version` as well. 

Also, update the minimum version of OD required as seen in https://github.com/WordPress/performance/commit/fb7e35c6c0d8e818cb91273fedeaeb84e3d0619b.